content/applications/sales/point_of_sale/configuration/epos_printers.rst

```diff
@@ -6,20 +6,53 @@ ePOS printers are designed to work seamlessly with Point of Sale systems. Once c
 devices automatically share information, allowing for direct printing of tickets from the POS system
 to the ePOS printer.
 
-Configuration
-=============
+Printer configuration
+=====================
+
+To use an ePos printer in Point of Sale, first set it up to get a certified domain that allows
+printing from a browser.
+
+1. Access the printer homepage by entering its IP address in a web browser.
+2. Click the :guilabel:`Advanced Settings` menu.
+3. Click :guilabel:`Administrator Login` in the top-right corner to log in.
+4. Click the :guilabel:`Network Security` tab, then go to :menuselection:`SSL/TLS --> Automatic Certificate Update`.
+5. Enable the :guilabel:`Automatic Update of CA-signed certificates` and :guilabel:`Update time`
+   options, set a time if needed, then click :guilabel:`Next` to let the printer restart.
+
+.. image:: epos_printers/auto_cert_update.png
+   :alt: Enable Automatic Certificate Update on the Epson ePos printer
 
-To use an ePos printer in Point of Sale:
+6. Once the printer has restarted, click the :guilabel:`Device Management` tab,
+   then go to :menuselection:`Date and Time --> Time Server`.
+7. Select :guilabel:`Use` in the :guilabel:`Use Time Server` field, set the
+   :guilabel:`Time Server Address` field to `be.pool.ntp.org` (or any valid time server),
+   and set the :guilabel:`Update Interval (min)` field to `10`.
 
-#. :ref:`Access the POS settings <configuration/settings>`.
-#. Activate the :guilabel:`ePos Printer` feature.
-#. Fill in the field with your ePos IP address.
+.. image:: epos_printers/time_server.png
+   :alt: Enable Time Server on the Epson ePos printer
+
+8. Click on :guilabel:`Ok` to let the printer apply the configuration.
```

Comment:
```suggestion
8. Click :guilabel:`Ok` to let the printer apply the configuration.
```
"Click" and "click on" are both correct, but to be consistent, I removed the "on" here.